documentation/src/docs/wire-protocol.md

```diff
@@ -11,102 +11,97 @@ You may obtain a copy of the License at
 
 This page describes the proposed Wire Protocol for the Streaming Service. See parent page for description of the service as a whole.
 
-Protocol
-========
+# Protocol
 
-Data is sent over the wire in self-contained "messages" that are either "requests" (messages from the client to the server) or "replies" (which are in response to a request and go back to the client).
+Data is sent over the wire in self-contained "messages" that are either "requests" (messages sent from the client to the server) or "replies" (responses sent from the server to the client).
 
-All requests and replies have an 8 byte header with two fields, (All data is written in BigEndian format):
-1.  Message Type - An integer (4 bytes) identifies the message type and this determines what fields will follow. (Note the protocol can be extended by adding new types)
-2.  Length - Unsigned integer 4 bytes (Messages should be &lt; 2^24, but the upper bits remain zero). How many bytes of data from this point forward are part of this message. (Possibly zero, indicating there is no data)
+All the requests and replies have an 8 bytes header with two fields.(All data is written in **BigEndian** format).
 
+- **Message Type**: An Integer of 4 bytes identifies the message type and determines what fields will follow. (Note that the protocol can be extended by adding new types.)
```

Comment:
@tkaitchuck `An Integer of 4 bytes identifies the message type and determines what fields will follow. `
Can i rephrase to: An Integer of 4 bytes identifies the message type and determines the remaining or remainder fields.

Reply:
An Integer of 4 bytes identifies the message type and determines the subsequent fields.

Reply:
@andreipaduroiu Accepted suggestion. Will  edit in all ocurrences